nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -65,12 +63,12 @@ def __init__(
         no_ddp_communication_hook: bool = False,
         **kwargs: Union[Any, Dict[str, Any]],
     ) -> None:
+        super().__init__(parallel_devices, num_nodes, cluster_environment, checkpoint_io, sync_batchnorm, **kwargs)
```

Comment:
Huh ?? Why these changes ?

Reply:
huh great catch, I dont know how these changes crept in. Probably when I rebased from r1.6.0 to main. Removed it :)

Reply:
Hehe